packages/slice-machine/lib/builders/SliceBuilder/SetupDrawer/index.tsx

```diff
@@ -91,18 +92,20 @@ const SetupDrawer: React.FunctionComponent<SetupDrawerProps> = ({
             )}
           </Flex>
         </Flex>
-        <HelpSection />
+        {!!storybook === false &&
```

Comment:
I think the check on next or nuxt is not necessary because we open the drawer only for those frameworks.
Maybe there is an array of supported frameworks somewhere to would allow specifying next or nuxt each time.

```suggestion
        { storybook ? <StorybookSection framework={framework} /> : null }
```